Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1101,12 +1101,12 @@ void USpatialNetDriver::AddActorChannel(const Worker_EntityId& EntityId, USpatia
 
 USpatialActorChannel* USpatialNetDriver::GetActorChannelByEntityId(const Worker_EntityId& EntityId) const
 {
-	USpatialActorChannel* const* ActorChannel = EntityToActorChannel.Find(EntityId);
+	USpatialActorChannel* ActorChannel = EntityToActorChannel.FindRef(EntityId);
```

Comment:
I meant replace the entire function with `return EntityToActorChannel.FindRef(EntityId);`
I've made this change in my branch actually, so you can just revert these changes.